### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by trisharia**
Added information for passing custom properties to vRA catalog requests

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -56,8 +56,9 @@ Other options that you can set include:
...
- * **private_key_path**: path to the SSH private key to use when logging in. Defaults to '~/.ssh/id_rsa' or '~/.ssh/id_dsa', preferring the RSA key. Only applies to instances where SSH transport is used (i.e. does not apply to Windows hosts with the WinRM transport configured).
+ * **private_key_path**: path to the SSH private key to use when logging in. Defaults to '~/.ssh/id_rsa' or '~/.ssh/id_dsa', preferring the RSA key. Only applies to instances where SSH transport is used; i.e., does not apply to Windows hosts with the WinRM transport configured.
  * **use_dns**: Defaults to `false`.  Set to `true` if vRA doesn't manage vm ip addresses.  This will cause kitchen to attempt to connect via hostname.
+ * **extra_parameters**: an array of other data to set on a catalog request, most notably custom properties. Allows updates to existing properties on the blueprint as well as the addition of new properties. The vRA REST API expects 'provider-' appended to the front of a property name, while the kitchen driver requires both the data type and set the value; e.g., set a property named 'mycustompropname' with `provider-mycustompropname: {type: "string",value: "mycustompropval"}`
```
Comment (adamleff) [inline README.md]: `extra_parameters` is actually a hash.  Each key in the hash is the property name, and the value is a another hash containing the value type and the value itself.

File: README.md [hunk 2]
```diff
@@ -70,10 +71,16 @@ platforms:
   - name: small
     driver:
       catalog_id: 8a189191-fea6-43eb-981e-ee0fa40f8f57
+      extra_parameters:
+        provider-mycustompropname: {type: "string",value: "smallvalue"}
```
Comment (adamleff) [inline README.md]: While this notation is technically correct and will work, the more "proper" way to notate nested hashes in YAML is like this:

```yaml
extra_parameters:
  provider-mycustompropname:
    type: string
    value: smallvalue
  provider-Vrm.DataCenter.Location:
    type: string
    value: Non-Prod
```

And you don't need to quote strings in YAML :)

**Comment by adamleff** [comment]
@trisharia, welcome to GitHub, and thanks for submitting your first Pull Request!! :100: 

I made two small comments: one to correct a small technical inaccuracy (hash vs. array) for the extra_parameters config property, and one to give you a more YAML-like way of specifying the parameters themselves.  Could you please make those changes and push them back up to your branch?  Once done, I'll merge it in!

Thanks again!

**Comment by trisharia** [comment]
Done. If you catch that I've typo'd anything, just let me know and I can update again. Thank you!

**Comment by adamleff** [comment]
This is awesome, @trisharia!  Thanks again.  I'm merging your change to master now. :+1:

**Pull request merged**